Core/Pixel.swift

```diff
@@ -190,6 +190,7 @@ public enum PixelName: String {
     
     case webKitDidTerminate = "m_d_wkt"
 
+    case settingsIconChangeFailed = "ms_icf"
```

Comment:
If it's a debug pixel, we typically prefix it with `m_d_`

Reply:
Thanks, good to know :)